api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -124,15 +124,11 @@ message GetAccessListMemberRequest {
 // UpsertAccessListMemberRequest is the request for upserting an access list member.
 message UpsertAccessListMemberRequest {
   // Renamed the "name" field to "member_name".
-  reserved 2;
-  reserved "name";
+  reserved 1, 2, 3;
+  reserved "access_list", "name", "reason";
 
-  // access_list is the name of access list.
-  string access_list = 1;
-  // member_name is the name of the user to add.
-  string member_name = 4;
-  // reason is the reason the user is being added.
-  string reason = 3;
```

Comment:
Is this field deletion going to cause problems to existing clients, or is this all new too?

Reply:
It's all new. It shouldn't cause any issues.

Reply:
Thank you.